bucket/cormanlisp.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": "3.1.2b",
+    "description": "Corman Lisp development environment for Common Lisp",
+    "homepage": "https://github.com/sharplispers/cormanlisp",
+    "license": "MIT",
+    "url": "https://github.com/sharplispers/cormanlisp/releases/download/v3.1.2b/CormanLisp-3.1.2b.zip",
+    "hash": "8bf722bd919a69ff743e2ae6448fcb3ccbc8be8b3ef746e6acc5cdcb1853ea95",
+    "extract_dir": "Corman Lisp",
+    "bin": [
+        "clboot.exe",
+        "clbootapp.exe",
+        "clconsole.exe",
+        "clconsoleapp.exe",
+        "CormanLisp.exe"
```

Comment:
There is no reason to keep this in path since it has no command line handler

```suggestion
```

Reply:
Thanks, missed that originally. (That's what you get for packaging at 2:00 AM.)